### PR TITLE
feat: seed genres, production companies and countries

### PR DESCRIPTION
**Pull request by fkula5**
set language code 'abc' as a placeholder

### CODE REVIEW

File: database/seeders/MainSeeder.php [hunk 1]
```diff
@@ -27,6 +31,25 @@ public function run()
             $mediumData = get_object_vars($x->medium);
             $mediumModel = Medium::firstOrCreate($mediumData);
 
+            //genre
+            $genreData = $x->genre;
+            foreach ($genreData as $genre){
+                $genreModel = Genre::firstOrCreate(['name' => $genre->name]);
```
Comment (marioooo0o) [inline database/seeders/MainSeeder.php:37]: You either add a new genre or fetch a genre named $genre->name its ok, but You don't make a relationship between genre and medium.
Put simply, you fill the genres table but don't add anything to the table genre_medium :( 
It is very important!

here's a tip:
[Dokumentacja](https://laravel.com/docs/9.x/eloquent-relationships#inserting-and-updating-related-models)

File: database/seeders/MainSeeder.php [hunk 2]
```diff
@@ -27,6 +31,25 @@ public function run()
...
+            }
+
+            //production companies
+            $productionCompanyData = get_object_vars($x->production_companies);
+            $productionCompanyModel = ProductionCompany::firstOrCreate($productionCompanyData);
```
Comment (marioooo0o) [inline database/seeders/MainSeeder.php:43]: same problem as above. Table compan_medium is empty now

File: database/seeders/MainSeeder.php [hunk 3]
```diff
@@ -27,6 +31,25 @@ public function run()
...
+
+            //countries
+            $countriesData = $x->countries;
+            foreach ($countriesData as $country){
+                $countryModel = Country::firstOrCreate([
+                    'code'=> 'abc',
```
Comment (marioooo0o) [inline database/seeders/MainSeeder.php]: why code is static 'abc'?
please change in action-movie.json code compatible with countries.json

**Review by marioooo0o — CHANGES_REQUESTED** [review]
You aren't completing pivot tables

File: database/seeders/MainSeeder.php [hunk 4]
```diff
@@ -31,23 +31,26 @@ public function run()
...
             $productionCompanyData = get_object_vars($x->production_companies);
             $productionCompanyModel = ProductionCompany::firstOrCreate($productionCompanyData);
+            $mediumModel->porduction_companies()->attach($productionCompanyModel->id);
 
-            //countries
+            //countries and country_medium
             $countriesData = $x->countries;
             foreach ($countriesData as $country){
                 $countryModel = Country::firstOrCreate([
                     'code'=> 'abc',
```
Comment (marioooo0o) [inline database/seeders/MainSeeder.php]: you should check name and code
'code' => $country->code,

**Review by marioooo0o — CHANGES_REQUESTED** [review]
very well created relationship in pivot!!
One small change and we merge to main

File: database/seeders/MainSeeder.php (hunk 4 above)
Comment (fkula5) [reply database/seeders/MainSeeder.php]: Oh well, i had created field in json file and stil left it like this

**Review by marioooo0o — APPROVED** [review]
All is ok 😄

**Pull request merged**